src/cdk-experimental/ui-patterns/behaviors/list-selection/list-selection.spec.ts

```diff
@@ -12,22 +12,22 @@ import {ListSelectionItem, ListSelection, ListSelectionInputs} from './list-sele
 import {ListNavigation, ListNavigationInputs} from '../list-navigation/list-navigation';
 
 describe('List Selection', () => {
-  interface TestItem extends ListSelectionItem {
+  interface TestItem<V> extends ListSelectionItem<V> {
     disabled: WritableSignalLike<boolean>;
   }
 
-  function getItems(length: number): SignalLike<TestItem[]> {
+  function getItems<V>(values: V[]): SignalLike<TestItem<V>[]> {
     return signal(
-      Array.from({length}).map((_, i) => ({
+      values.map((value, i) => ({
         index: signal(i),
-        id: signal(`${i}`),
+        value: signal(value),
         disabled: signal(false),
         isAnchor: signal(false),
       })),
     );
   }
 
-  function getNavigation<T extends TestItem>(
+  function getNavigation<T extends TestItem<V>, V>(
```

Comment:
This feels awkward, can't it just be generic on `V` and then `items: SignalLike<TestItem<V>[]>` and `args: Partial<ListNavigationInputs<TestItem<V>>>`

Reply:
Oh nvm, I think maybe that says something a little bit different than this